systems/WikibaseClient/03-Context_and_Scope.md

```diff
@@ -21,6 +21,6 @@ WikibaseClient is a "plugin" (= extension) that lives inside [MediaWiki](../over
 | [ChangeHandler](./05-Building_Block_View.md#entity-change-notifications)      | Receives notifications to update and purge pages when an Item is changed on the [Repo](../overview/12-Glossary.md#wikibase-repository)                                                                        |
 | [UpdateRepoHookHandler](./05-Building_Block_View.md#linked-site-page-changes) | Notifies the [Repo](../overview/12-Glossary.md#wikibase-repository) to update the connected Item when a page is moved or deleted                                                                              |
 | [Data Bridge](./05-Building_Block_View.md#client-side-item-edits)             | A frontend component enabling [Repo](../overview/12-Glossary.md#wikibase-repository) edits on the Client via the Repo API                                                                                     |
-| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data                                                                                                                                                                  |
+| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data. For a full description see [Special Pages](../overview/12-Glossary.md#special-page)                                                                             |
```

Comment:
For extra clarity about what to expect I think this should be reworded to include text saying it's linking you to the glossary for a general definition of the term special page. When reading it now it seems slightly confusing as there are 2 textual links with the text "Special Pages" that link to different places without saying which goes where.

Not sure of the best text, but even (See what a Special Pages is in the glossary) or something might make sense.

Reply:
added